client/my-sites/stats/info-panel/index.jsx

```diff
@@ -238,7 +238,7 @@ class StatsInfoPanel extends React.PureComponent {
 									href="http://wordpress.com/support/category/users/"
 								>
 									<Gridicon icon="folder" />
-									{ this.props.translate( 'About Users' ) }
+									{ this.props.translate( 'About users' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.